### ISSUE TITLE
Handle IO errors in the load command

### ISSUE THREAD

**Issue by hammady** [Qatar-Computing-Research-Institute/NADEEF#34]
:> load sldkfsdls
Oops, something is wrong. Please check the log in the output dir.
Exception: java.io.FileNotFoundException: null

**Comment by zyzyis** [comment]
It is correctly handled right?

**Comment by hammady** [comment]
It prints the IOException message, while it is better to wrap it in a more
user friendly message.

**Comment by zyzyis** [comment]
of course, this has a way old discussion back with Ihab, where we think no grandma is going to use console. 
Grandma needs to use dashboard.
